patches/@vue__runtime-core@3.3.4.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/dist/runtime-core.esm-bundler.js b/dist/runtime-core.esm-bundler.js
+index 860d66fd0429aae7b94a0aa2b4e35ecfb06546e4..48648328b68cc49e3fad49ef829d474120c590ee 100644
+--- a/dist/runtime-core.esm-bundler.js
++++ b/dist/runtime-core.esm-bundler.js
+@@ -1152,17 +1152,21 @@ function patchSuspense(n1, n2, container, anchor, parentComponent, isSVG, slotSc
+   if (pendingBranch) {
+     suspense.pendingBranch = newBranch;
+     if (isSameVNodeType(newBranch, pendingBranch)) {
+-      patch(
+-        pendingBranch,
+-        newBranch,
+-        suspense.hiddenContainer,
+-        null,
+-        parentComponent,
+-        suspense,
+-        isSVG,
+-        slotScopeIds,
+-        optimized
+-      );
++      // When the previous pending is not yet resolved (e.g. double click the link),
++      // the `pendingBranch.el` is not yet fulfilled, will causing the `patch` to fail.
++      if (pendingBranch.el) {
++        patch(
++          pendingBranch,
++          newBranch,
++          suspense.hiddenContainer,
++          null,
++          parentComponent,
++          suspense,
++          isSVG,
++          slotScopeIds,
++          optimized
++        );
++      }
+       if (suspense.deps <= 0) {
+         suspense.resolve();
+       } else if (isInFallback) {
+@@ -6119,6 +6123,9 @@ function baseCreateRenderer(options, createHydrationFns) {
+   const move = (vnode, container, anchor, moveType, parentSuspense = null) => {
+     const { el, type, transition, children, shapeFlag } = vnode;
+     if (shapeFlag & 6) {
++      // This I am not yet sure why, but it can be undefined in some conditions
++      if (!vnode.component)
++        return
```

Comment:
I am able to fix the bug by patching Vue with these two changes. I am thinking of sending PRs to Vue core but can't find a good way to reproduce it with pure Vue, nor do I find a rational reason for them.

Also not sure how feasible it is for Nuxt to ship a patched Vue or so

Reply:
We were able to solve this without any patches by disabling transitions in nuxt.

If this applies to this exact issue as well, https://github.com/vuejs/core/pull/9309 should fix this upstream